lib/win-releases.js

```diff
@@ -50,20 +50,7 @@ function parseRELEASES(content) {
             var filename = parts[2];
             var isDelta = filename.indexOf('-full.nupkg') == -1;
 
-            var filenameParts = filename
-                .replace(".nupkg", "")
-                .replace("-delta", "")
-                .replace("-full", "")
-                .split(/\.|-/)
-                .reverse();
-
-            var version = _.chain(filenameParts)
-                .filter(function(x) {
-                    return /^\d+$/.exec(x);
-                })
-                .reverse()
-                .value()
-                .join('.');
+            var version = filename.split('-').slice(1,-1).join('-');
```

Comment:
In the existing code Nuts is excluding any non-digit characters from the version string when it reconstructs it.  This means that you don't get the complete string to compare against the local one.  Though, actually, if we leave the local build strings untouched then maybe it's not such a big deal.  Hang on a second, lemme check something.